modules/oe_theme_content_project/config/overrides/core.entity_view_display.node.oe_project.default.yml

```diff
@@ -119,6 +119,14 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  oe_departments:
+    type: skos_concept_entity_reference_label
+    weight: 6
+    region: content
+    label: above
+    settings:
+      link: true
```

Comment:
I think this should not be a clickable link, is it?

Reply:
Fixed